### ISSUE TITLE
Code style for contributions not documented

### ISSUE THREAD

**Issue by mike182uk** [gitevents/gitevents-tito#8]
This repo is currently monitored by `hound-ci` but there is no documentation indicating what code style rules are in force. Either `hound-ci` should be disabled for this repo, or it should be documented what code style rules the contributor should use and the facility to run these style checks locally should be provided

**Comment by PatrickHeneise** [comment]
hound has been deactivated. Code style for gitevents can be found there, you can use whatever you want but consistency with gitevents is appreciated.

**Comment by mike182uk** [comment]
Ok cool. Is it possible to tell hound on a per repo basis what code style rules to use? I personally think we should keep it enabled, and provide a code-style for contributors to follow as a first step.

**Comment by PatrickHeneise** [comment]
Yeah, you can use the .hound.yml to configure, hound uses mainly .jscs, .jshintrc etc.

**Comment by iancrowther** [comment]
can we create a CONTRIBUTING.md and re-use?
